### DISCUSSION TITLE
Unexpected location of ccache cache directory

### DISCUSSION THREAD

**Discussion by mtsokol** [ccache/ccache#1527] [Questions & answers] [answered]
Hi!

The `ccache` docs say that:
```
cache_dir (CCACHE_DIR)

    This option specifies where ccache will keep its cached compiler outputs.

    On non-Windows systems, the default is $HOME/.ccache if such a directory exists, otherwise $XDG_CACHE_HOME/ccache if XDG_CACHE_HOME is set, otherwise $HOME/Library/Caches/ccache (macOS) or $HOME/.config/ccache (other systems).
```

I'm using a linux machine so I expect to find the cache directory either in `$HOME/.ccache` or `$HOME/.config/ccache` but it looks that it ended up in `$HOME/.cache/ccache` on my machine.

Is it due to some extra configuration that I might have on my machine or are docs out-of-date?

I have `ccache` version `4.10.1`

**Comment by jrosdahl** [comment, marked as answer]
It's a typo in the documentation. Fixed in a9cd717bcf9b568671e8c33d0fb5089f5ff87b6a. Thanks for noticing!

**Reply by mtsokol** [reply]
Thank you!
